src/RoadRegistry.BackOffice/Core/AttributeHash.cs

```diff
@@ -5,64 +5,93 @@ namespace RoadRegistry.BackOffice.Core
 
     public readonly struct AttributeHash : IEquatable<AttributeHash>
```

Comment:
AttributeHash was horribly broken - hashing is not a commutative operation and we should be able to replace parts of what is being hashed.